src/platform/user/tests/authentication/utilities.unit.spec.js

```diff
@@ -115,3 +105,33 @@ describe('authentication URL helpers', () => {
     expect(global.window.location).to.include('/v1/sessions/verify/new');
   });
 });
+
+describe('standaloneRedirect', () => {
+  beforeEach(fakeWindow);
+  afterEach(() => {
+    global.window = oldWindow;
+  });
+
+  it('should return null when an application param is not provided', () => {
+    global.window.location.search = '';
+    expect(standaloneRedirect()).to.equal(null);
+  });
+
+  it('should return null when an application redirect is not found', () => {
+    global.window.location.search = '?application=unmappedapplication';
+    expect(standaloneRedirect()).to.equal(null);
+  });
+
+  it('should return an plain url when no "to" search query is provided', () => {
+    global.window.location.search = '?application=myvahealth';
+    expect(standaloneRedirect()).to.equal(externalRedirects.myvahealth);
+  });
+
+  it('should strip any CRLF characters from the "to" parameter', () => {
```

Comment:
How do CRLF characters end up in the parameter (a URL path)?

Reply:
This was implemented based on advice from: https://cheatsheetseries.owasp.org/cheatsheets/Unvalidated_Redirects_and_Forwards_Cheat_Sheet.html

I.e. malicious entities injecting a URL parameter that leads to them being able to inject headers by splitting the URL in the HTTP request with a CRLF.